cellprofiler/modules/combineobjects.py

```diff
@@ -172,7 +175,7 @@ def display(self, workspace, figure):
         )
 
     def combine_arrays(self, labels_x, labels_y):
-        output = numpy.zeros_like(labels_x)
+        output = numpy.zeros_like(labels_x).astype("uint16")
```

Comment:
Type casting is definitely not necessary here. `zeros_like` will copy the dtype of the input, which you already set.

Reply:
Yep, agreed